packages/video_player/video_player/test/video_player_test.dart

```diff
@@ -337,6 +401,20 @@ void main() {
         expect(platformEx.code, equals('VideoError'));
       });
 
+      test(
+          'when controller is initialized with invalid url it should throw VideoError',
```

Comment:
For tests that aren't directly testing the constructor don't need to be duplicated, so instead of adding this and the error-clearing one, you can just change the existing version to use the new API.